docs/acceptance/db.md

```diff
@@ -0,0 +1,65 @@
+## Database Deliverable Acceptance Test 
+
+### Description of Deliverable
+
+This deliverable includes the implementation of an ETL process, the design of a database schema, the selection of a format for raw data storage, infrastructure as code, database configuration, and accompanying documentation.
+
+The acceptance test below provides steps to verify that the deliverable meets our agreed-upon specifications.
+
+### Input Data
+
+The input data is stored in Parquet format on AWS S3 (object storage), specifically in the file `space2stats_updated.parquet`. Any additional fields must be appended to this file. The Parquet file is tabular with the following columns:
+- `hex_id`
+- `{aggregation_method[sum, mean, etc.]}_{variable_name}_{year}`
```

Comment:
Small comment, but I would find it easier to store variables with this convention `{variable_name}_{aggregation_method[sum, mean, etc.]}_{year}`, I think it will be easier to read/group once we have several variables.

Reply:
changed naming convention in doc.